bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/BaseThingHandler.java

```diff
@@ -77,10 +82,12 @@
     /**
      * Creates a new instance of this class for the {@link Thing}.
      *
-     * @param thing
-     *            thing
+     * @param thing the thing that should be handled, not null
+     *
```

Comment:
The empty lines are not required, but from my point of view it makes the javadoc/code more readable.